ChangeLog

```diff
@@ -34,6 +34,16 @@ Bugfix
      i386 with SSE2. Found by László Langó. Fixes #1550
    * Fix namespacing in header files. Remove the `mbedtls` namespacing in
      the `#include` in the header files. Resolves #857
+   * Fix decryption of zero length messages (all padding) in some circumstances:
+     DTLS 1.0 and 1.2, and CBC ciphersuites using encrypt-then-MAC. Most often
```

Comment:
Blocker: This reads as if there was a problem whenever DTLS is used, _or_ when TLS is used with CBC ciphersuites and EtM. Suggestion: `Fix decryption of zero length messages (all padding) when a CBC ciphersuite together with Encrypt-then-MAC is used.`
Also, we should add a note on the previous, erroneous behaviour: `Previously, such a message was wrongly reported as an invalid record and therefore lead to connection termination in TLS.`

Reply:
This should be resolved in gatekeeping.